lm/lm_image.py

```diff
@@ -0,0 +1,120 @@
+__all__ = ["LmImageJob"]
+
+import shutil
+
+from sisyphus import *
+from i6_core import rasr
+from i6_core import util
+
+Path = setup_path(__package__)
+
+
+class LmImageJob(rasr.RasrCommand, Job):
```

Comment:
This Job does not have an explicit hash function, therefore all parameters of the `__init__` function will be included in the hash, including `extra_post_config`.

But also you would not want to hash all entries of the crp but only those relevant to the LM image.